dockerclient.go

```diff
@@ -738,3 +738,242 @@ func (client *DockerClient) BuildImage(image *BuildImage) (io.ReadCloser, error)
 	uri := fmt.Sprintf("/%s/build?%s", APIVersion, v.Encode())
 	return client.doStreamRequest("POST", uri, image.Context, headers)
 }
+
+// Networks
+
+func (client *DockerClient) ListNetworks(name string, partialId string) ([]Network, error) {
+	uri := fmt.Sprintf("/%s/networks", APIVersion)
+
+	if name != "" {
+		uri += "&name=" + name
```

Comment:
We support only one query parameter for these endpoints in libentworks, either name or partial-id
I think this should be: `uri += "?name=" +name` and `uri += "?partial-id=" + partialId` below.